### PR TITLE
(fix) exclude ATs from sonar cloud

### PR DESCRIPTION
**Pull request by frankisawesome**
Signed-off-by: Frank Li <b439988l@gmail.com>

## PR description
Excludes ATs from sonar cloud runs

## Fixed Issue(s)
fixes https://github.com/hyperledger/besu/issues/3142

### CODE REVIEW

**Comment by frankisawesome** [comment]
Ok so sonar cloud on 8f02b9c flags code in the acceptance-tests folder that doesn't directly run tests as uncovered code.

**Comment by frankisawesome** [comment]
Nothing flagged after adding the config option

**Comment by frankisawesome** [comment]
Note to myself while I remember - we also need a to test that coverage
works for non AT files (don't think that'd be an issue but just in case)

**Review by jflo — APPROVED** [review]

File: ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/BlockchainModule.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ MutableWorldView getMutableWorldView(
       final GenesisState genesisState,
       @Named("KeyValueStorageName") final String keyValueStorageName) {
     if ("memory".equals(keyValueStorageName)) {
+      System.out.println("trigger bad coverage");
```
Comment (macfarla) [inline ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/BlockchainModule.java:64]: we want to remove this line before merging

**Review by macfarla — DISMISSED** [review]

File: ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/BlockchainModule.java (hunk 1 above)
Comment (frankisawesome) [reply ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/BlockchainModule.java:64]: Oh yeah sorry I requested your review and then realised there's one last thing I needed to test still😛

**Comment by frankisawesome** [comment]
Ok I've tested:
1. File under acceptance-tests with no tests, coverage tested (not good)
2. File under acceptance-tests with no tests, new config, coverage not tested (what we want)
3. File outside of AT, new config, coverage still tested (good)

**Pull request merged**